examples/selection_tools/unit_circle.py

```diff
@@ -0,0 +1,114 @@
+"""
+Unit circle
+===========
+
+Example with linear selectors on a sine and cosine function that demonstrates the unit circle.
+
+This shows how fastplotlib supports bidirectional events, drag the linear selector on the sine
+or cosine function and they will both move together.
+
+Click on the sine or cosine function to set the colormap transform to illustrate the sine or
+cosine function output values on the unit circle.
+"""
+
+# test_example = false
+# sphinx_gallery_pygfx_docs = 'screenshot'
+
+
+import numpy as np
+import fastplotlib as fpl
+
+
+# helper function to make a cirlce
+def make_circle(center, radius: float, n_points: int) -> np.ndarray:
+    theta = np.linspace(0, 2 * np.pi, n_points)
+    xs = radius * np.cos(theta)
+    ys = radius * np.sin(theta)
+
+    return np.column_stack([xs, ys]) + center
+
+
+# create a figure with 3 subplots
+figure = fpl.Figure((3, 1), names=["unit circle", "sin(x)", "cos(x)"], size=(700, 1024))
```

Comment:
does the gallery like this figure size? I thought it had to be something else to appear correctly in the docs?

Reply:
Extra height upto 1024 is fine (texture size limit we imposed). Width other than 700 causes weirdness on docs